tools/scripts/install-sdk-preview.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs')
```

Comment:
If you have a JS for it, maybe we can replace this part https://github.com/cowprotocol/cow-sdk/blob/main/.github/workflows/publish-github-packages.yml#L106 

Unrelated, and please ignore because its on me, but if we are moving out some scripts from the CI, this one is big and hard to mantain https://github.com/cowprotocol/cow-sdk/blob/main/.github/workflows/publish-github-packages.yml#L214